src/main/java/net/juniper/netconf/element/Hello.java

```diff
@@ -72,7 +72,7 @@ public static Hello from(final String xml)
             builder.capability(node.getTextContent());
         }
         final Hello hello = builder.build();
-        log.info("hello is: {}", hello.getXml());
+        if (log.isDebugEnabled()) log.debug("hello is: {}", hello.getXml());
```

Comment:
If this project switched to Log4j2, we could use suppliers as the arguments into `log.debug`. By doing this, no need for this `if` statement at the start. If debug is not enabled, the supplier will never get called.

See https://logging.apache.org/log4j/2.x/log4j-api/apidocs/org/apache/logging/log4j/Logger.html#debug-java.lang.String-org.apache.logging.log4j.util.Supplier...-

Reply:
I agree, but at the moment this project does not use Log4j2. The point of this PR is to move logging from "Info" to "Debug".

Reply:
Curious on estimate of how much work it would be to switch to Log4j2?